packages/sources/lido-por/src/endpoint/reserve.ts

```diff
@@ -0,0 +1,41 @@
+import { InputParameters } from '@chainlink/external-adapter-framework/validation'
+import {
+  PoRProviderEndpoint,
+  PoRProviderResponse,
+} from '@chainlink/external-adapter-framework/adapter/por'
+import { config } from '../config'
+import { balanceTransport } from '../transport/reserve'
+
+export const inputParameters = new InputParameters(
+  {
+    withdrawalCredential: {
+      required: true,
+      type: 'string',
+      description: "Validator's withdrawal credential",
+    },
+    bufferContract: {
```

Comment:
Maybe `lidoContract` or `stETHContract` is better as it contains way more things and not just a buffer for ETH